plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -342,7 +346,7 @@ void updateRunnerButtons(MajorVersion majorVersion) {
    * Asynchronously updates the project hierarchy.
    */
   private void updateHierarchy(final MajorVersion majorVersion) {
-    Job job = new Job("Update Hierarchy") {
+    Job job = new Job("Update Hierarchy") { //$NON-NLS-1$
```

Comment:
This string likely should be externalized as it is reported on the Progress View.